pkgs/top-level/python-packages.nix

```diff
@@ -8131,6 +8176,30 @@ in modules // {
     };
   };
 
+  django_raster = buildPythonPackage rec {
+    name = "django-raster-${version}";
+    version = "0.1.7a1";
```

Comment:
Yes. `0.1.6` even if published is dysfunctional. To get something working based on this, we have had to use the master branch, not the last release.